apps/server/src/shared/repo/base.do.repo.ts

```diff
@@ -1,4 +1,4 @@
-import { EntityDTO, EntityName, FilterQuery, wrap } from '@mikro-orm/core';
+import { EntityName, FilterQuery } from '@mikro-orm/core';
```

Comment:
In generally i like if we restructure the code in a way that the logic inside the map`s for update, delete, create are outsource to private functions and that we not directly return logic operations.